packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -150,7 +152,7 @@ const WithLabel = ({
           field.type !== "multiemail" &&
           field.label && (
             <div className="mb-2 flex items-center">
-              <Label className="!mb-0 flex">
+              <Label className="!mb-0 flex" htmlFor={htmlFor}>
```

Comment:
Added htmlFor to correctly connect with input. Fixes accessiblity issue discovered during tests.